database/migrations/2019_12_16_151408_create_uploads_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
PR looks perfect except for this. we need to use the documents table to store all user uploaded files.

the exception would be the avatar which is stored on the user itself, and also the company_logo which is stored in the settings tree.